.github/workflows/updoc.yml

```diff
@@ -0,0 +1,61 @@
+name: Updoc
```

Comment:
We may rename this workflow file `provider-updoc.yml` to be compatible with the other reusable workflow definitions.